ArjunaJTS/integration-jakarta/pom.xml

```diff
@@ -0,0 +1,453 @@
+        <artifactId>nexus-staging-maven-plugin</artifactId>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifestEntries>
+              <Specification-Title>JTA/JTS</Specification-Title>
+              <Specification-Version>1.2/1.0</Specification-Version>
+              <Specification-Vendor>Oracle</Specification-Vendor>
+              <Implementation-Version>${project.version} (revision: ${buildNumber})</Implementation-Version>
+              <arjuna-scm-revision>${buildNumber}</arjuna-scm-revision>
+              <arjuna-properties-file>jbossts-properties.xml</arjuna-properties-file>
+              <arjuna-builder>JBoss Inc. [${user.name}] ${os.name} ${os.version} ${buildproperty.date}</arjuna-builder>
+            </manifestEntries>
+          </archive>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-test</id>
+            <phase>test</phase>
+            <goals>
+              <goal>test</goal>
+            </goals> 
+            <configuration>
+              <useManifestOnlyJar>false</useManifestOnlyJar>
+              <skip>true</skip>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>buildnumber-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>validate</phase>
+            <goals>
+              <goal>create</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <doCheck>false</doCheck>
+          <doUpdate>false</doUpdate>
+          <shortRevisionLength>5</shortRevisionLength>
+          <revisionOnScmFailure>${project.version}-noscm</revisionOnScmFailure>
+        </configuration>
+      </plugin>
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>transform-jar</id>
+            <phase>package</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <target>
+                <property name="plugin_classpath" refid="maven.plugin.classpath"/>
+                <java classname="org.eclipse.transformer.jakarta.JakartaTransformer">
+                  <arg value="${org.jboss.narayana.jts:narayana-jts-integration:jar}"/>
+                  <arg value="${project.build.directory}/${project.build.finalName}.jar"/>
+                  <arg value="-q"/>
+                  <arg value="-o"/>
+                  <arg value="-tr"/>
+                  <arg value="${project.basedir}/rules/jakarta-renames.properties"/>
```

Comment:
It could do. I will delete the directory` ${project.basedir}/../../jakarta-rules` and then reference the rules in the top level directory (note that I did this work in multiple commits and after a while stopped duplicating the rules file every time I needed it).